README.md

```diff
@@ -1,48 +1,207 @@
 [![Maven Central](https://maven-badges.herokuapp.com/maven-central/za.co.absa.spline/spline-core/badge.svg)](https://maven-badges.herokuapp.com/maven-central/za.co.absa.spline/spline-core)
 
 The project consists of three main parts:
 - Spark Agent that sits on drivers, capturing the data lineage from Spark jobs being executed by analyzing the execution plans
-- Rest Gateway, that receive the lineage data from agent and stores it in the database
-- Web UI application that visualizes the stored data lineages
+
+-   Rest Gateway, that receive the lineage data from agent and stores it in the database
+
+-   Web UI application that visualizes the stored data lineages
+
+![Spline diagram](https://user-images.githubusercontent.com/5530211/69061052-c4795400-0a18-11ea-9990-91836e2d648a.png)
+
 
 There are several other tools. Check the examples to get a better idea how to use Spline.
 
 Other docs/readme files can be found at:
-- [ClientUI](client-ui/README.md)
-- [Examples](examples/README.md)
-- [Spark Agent](spark/agent/README.md)
-- [Spline Paper](https://github.com/AbsaOSS/spline/releases/download/release%2F0.2.7/Spline_paper_IEEE_2018.pdf)
+-   [ClientUI](client-ui/README.md)
+
+-   [Examples](examples/README.md)
+
+-   [Spark Agent](spark/agent/README.md)
+
+-   [Spline Paper](https://github.com/AbsaOSS/spline/releases/download/release%2F0.2.7/Spline_paper_IEEE_2018.pdf)
 
 Spline currently supports Spark 2.2+, but in older versions (especially 2.2) lineage information provided by spark is limited.
 
 ## Motivation
 
 Spline aims to fill a big gap within the Apache Hadoop ecosystem. Spark jobs shouldn’t be treated only as magic black boxes; people should be able to understand what happens with their data. Our main focus is to solve the following particular problems:
 
-- Regulatory requirement for SA banks (BCBS 239)
+-   Regulatory requirement for SA banks (BCBS 239)
 
     By 2020, all South African banks will have to be able to prove how numbers are calculated in their reports to the regulatory authority.
 
-- Documentation of business logic
+-   Documentation of business logic
 
     Business analysts should get a chance to verify whether Spark jobs were written according to the rules they provided. Moreover, it would be beneficial for them to have up-to-date documentation where they can refresh their knowledge of a project.
 
-- Identification of performance bottlenecks
+-   Identification of performance bottlenecks
 
     Our focus is not only business-oriented; we also see Spline as a development tool that should be able to help developers with the performance optimization of their Spark jobs.
 
+---
+
+## Get Spline
+To get started, you need to get a minimal set of Spline's moving parts - 
+a server, an admin tool and a client Web UI to see the captured lineage.
+
+There are two ways how to do it:
+
+#### Download prebuild Spline artifacts from the Maven repo
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[heading-increment] Heading levels should increment by one level at a time](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4568241)